src/components/DestinationHeroSection.vue

```diff
@@ -0,0 +1,326 @@
+<template>
+  <div class="hero">
+    <h1><span>01</span>Pick your destination</h1>
+    <div class="hero-section">
+      <div>
+        <img :src="defaultImage" alt="" />
+      </div>
+      <div class="options">
+        <ul>
+          <li
+            v-for="planet in planets"
+            :key="planet.name"
+            @click="selectPlanet(planet)"
+            :class="{ selected: planet.name === selectedOption }"
+          >
+            {{ planet.name }}
+          </li>
+        </ul>
+        <div v-for="planet in planets" :key="planet.name">
+          <div v-if="selectedPlanet === planet.name">
+            <h2>{{ planet.name }}</h2>
+            <p class="option-detail">
+              {{ planet.description }}
+            </p>
+            <div class="measurements">
+              <div>
+                <h3>AVG. DISTANCE</h3>
+                <p>{{ planet.distance }}</p>
+              </div>
+              <div>
+                <h3>Est. travel time</h3>
+                <p>{{ planet.days }}</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  data() {
+    return {
+      planets: [
+        {
+          name: "MOON",
+          description:
+            "See our planet as you’ve never seen it before. A perfect relaxing trip away to help regain perspective and come back refreshed. While you’re there, take in some history by visiting the Luna 2 and Apollo 11landing sites.",
+          distance: "384,400 KM",
+          days: "3 DAYS",
+          img: require("@/assets/images/image/image-moon.webp"),
+        },
+        {
+          name: "MARS",
+          description:
+            "Don’t forget to pack your hiking boots. You’ll need them to tackle Olympus Mons, the tallest planetary mountain in our solar system. It’s two and a half times the size of Everest!",
+          distance: "225 MIL. KM",
+          days: "9 MONTHS",
+          img: require("@/assets/images/image/image-mars.webp"),
+        },
+        {
+          name: "EUROPA",
+          description:
+            "The smallest of the four Galilean moons orbiting Jupiter, Europa is a winter lover’s dream. With an icy surface, it’s perfect for a bit of ice skating, curling, hockey, or simple relaxation in your snug wintery cabin.",
+          distance: "628 MIL. KM",
+          days: "3 YEARS",
+          img: require("@/assets/images/image/image-europa.webp"),
+        },
+        {
+          name: "TITAN",
+          description:
+            "The smallest of the four Galilean moons orbiting Jupiter, Europa is a winter lover’s dream. With an icy surface, it’s perfect for a bit of ice skating, curling, hockey, or simple relaxation in your snug wintery cabin.",
+          distance: "1.6 BIL. KM",
+          days: "7 YEARS",
+          img: require("@/assets/images/image/image-titan.webp"),
+        },
+      ],
+      selectedPlanet: "MOON",
```

Comment:
This should be `selectedPlanet: this.planets[0]`
What this means is that, by default, selectedPlanet will be equals to the first planet in your `planets` array